api/cpp/esp-idf/slint/src/slint-esp.cpp

```diff
@@ -305,16 +325,38 @@ void slint_esp_init(slint::PhysicalSize size, esp_lcd_panel_handle_t panel,
                     std::optional<std::span<slint::platform::Rgb565Pixel>> buffer2,
                     slint::platform::SoftwareRenderer::RenderingRotation rotation)
```

Comment:
I wonder if we should remove the rotation in there since it was experimental only, and only have the rotation in the "complex" overload